stores-api/staking-api/src/main/java/com/bloxbean/cardano/yaci/store/api/staking/StakingApiConfiguration.java

```diff
@@ -5,10 +5,7 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ConditionalOnProperty(name = {"store.staking.enabled", "store.staking.api-enabled"},
-        havingValue = "true",
-        matchIfMissing = true
-)
+@ConditionalOnProperty(name = {"store.staking.enabled"}, havingValue = "true", matchIfMissing = true)
```

Comment:
@edridudi Thanks for PR.

If I understand correctly, you are attempting to use the service class from the API module, and that is why conditional properties are being added at the controller level.

However, the current approach is to maintain only a standard interface for the storage layer. The 'service' layer classes are specific to the controller and may change in the future. Otherwise, we would need to maintain two sets of interfaces and ensure they are backward compatible.

I believe the custom application should rely on the storage interface, which provides domain objects, and build its own service layer if necessary.

Alternatively, the custom application can depend on API endpoints.

But after reviewing your changes, I am now open to standardizing the service layer. However, we need to do the following:

1. Review all service layer methods.
2. Maintain the API level flag name at the store level. So, if api-enabled is false, both service and controllers are disabled. 
The api-enabled flag is at the store level, so it would always be ``store.<store_name>.api-enabled``.
3. Introduce another flag (or flags) for endpoints/controllers of the store API module, which would be at the controller level.

    There are two possible options:
    - **Option 1:** ``store.<store_name>.endpoints.<api_tag_name>.enabled = true``
    In this case, it's clear that the property will be defined in the specific store's starter.
    - **Option 2:** ``store.endpoint.<api_tag_name>.enabled = true``
    In this case, I am not sure what is the best starter to keep this property, as we have the same API tag in multiple stores.

    **Can we go with Option 1 for now because there is no ambiguity?** But we can explore Option 2 in future releases.